Dockerfile

```diff
@@ -0,0 +1,397 @@
+FROM debian:stretch
+
+SHELL ["/bin/bash", "-c"]
+
+WORKDIR /tmp
+
+# Build tools, and misc supporting tools
+RUN apt update && \
```

Comment:
I can't see any cleaning of the apt cache in the Dockerfile, yet there isn't any packages cached when I run it.
I've had to manually remove cached packages and temporaries in other dockerfiles.
Do you know why it doesn't seem to be needed in this one?